certified-connectors/Tesseron ASM Basic Data/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1085 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Tesseron ASM Basic Data",
+    "description": "This connector allows you to use Tesseron ASM Basic Data API to create, edit and search business partners and contacts. ",
+    "contact": {
+      "name": "Tesseron ASM Support",
+      "email": "development@llnet.de",
+      "url": "https://www.docs.tesseron.de"
+    },
+    "version": "1.0"
+  },
+  "host": "dynamichosturl",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [],
+  "paths": {
+    "/": {},
```

Comment:
@developmentLL empty operations are not allowed. Please define an operation here or remove it.
More information about operation definitions can be found in [coding standards](https://docs.microsoft.com/en-us/connectors/custom-connectors/coding-standards#definitions-and-parameters) and in [the documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions).